src/Microsoft.Identity.Web.TokenAcquisition/ServiceCollectionExtensions.cs

```diff
@@ -45,9 +44,9 @@ public static class ServiceCollectionExtensions
             bool forceSdk = !services.Any(s => s.ServiceType.FullName == "Microsoft.AspNetCore.Authentication.IAuthenticationService");
 #endif
 
-            if (services.FirstOrDefault(s => s.ImplementationType == typeof(ICredentialsLoader)) == null)
+            if (services.FirstOrDefault(s => s.ImplementationType == typeof(DefaultCertificateLoader)) == null)
```

Comment:
wouldn't want to test that there is no implementation of ICredentialLoader?
that is:

```csharp
if (services.FirstOrDefault(s => s.ServiceType== typeof(ICredentialsLoader)) == null)
{
  services.TryAddSingleton<DefaultCertificateLoader>();
}
```

?

Reply:
That is how I originally wanted to write it 😄 

I went with this as it is more consistent with the rest of the file.